rustls/tests/api.rs

```diff
@@ -5253,6 +5253,78 @@ fn test_server_accepts_client_with_no_ecpoints_extension_and_only_ffdhe_cipher_s
     assert!(server.process_new_packets().is_ok());
 }
 
+#[cfg(feature = "tls12")]
+#[test]
+fn test_server_avoids_cipher_suite_with_no_common_kx_groups() {
```

Comment:
should we also contain some negative cases?

Reply:
You mean a test where server should err with `NoKxGroupsInCommon`? This test already covers that: `test_server_rejects_clients_without_any_kx_group_overlap`.

Reply:
Oh, got it. Did not check unchanged parts. Thx for pointing out.